src/ConfigurationsProvider.tsx

```diff
@@ -24,6 +24,7 @@ const mergeDefaultConfiguration = (target: any, source: any) => {
 
 const defaultConfiguration = {
   appDesignSystem: defaultAppDesignSystemConfig,
+  filter: defaultFilterConfig,
```

Comment:
i would suggest calling this the way it is defined in main server now as `resourceFilter` to be always sure what these filters are matching on the server side. just `filter` is very generic

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/dcaa33b4bb11e65b236ddfc124ee08d93938c992